webapp/src/lib/components/spot-listings/spots-list-component.svelte

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+    import { ChargingStation, PlugFilled, Home } from 'carbon-icons-svelte';
+    import { TooltipIcon } from 'carbon-components-svelte';
+    import DetailModal from './detail-modal.svelte';
+    export let listing;
+
+    let modalOpen = false;
+
+    const handleClick = () => {
+        modalOpen = true;
+    };
+</script>
+
+<div
+    class="listing-info"
+    on:click={() => handleClick()}
+    on:keyup={() => {}}
+    role="button"
+    tabindex="0"
+>
+    <div class="address">
+        <p class="listing-head">{listing.location.street_address}</p>
+        <p>{listing.location.postal_code}</p>
+    </div>
+    <div class="features">
+        <div class="feature">
+            {#if listing.features.charging_station}
+                <TooltipIcon tooltipText="Charging Station Available" direction="right" align="end">
+                    <i><ChargingStation size={20} /></i>
+                </TooltipIcon>
+            {/if}
+        </div>
+        <div class="feature">
```

Comment:
Can you extract this feature into a component? I have a feeling that we can reuse it somewhere else (maybe in other type of listing)

Reply:
In my opinion we would never use just the icons for a listing in another place, it would be the whole component with the rest of the details as well. Increasing the granularity of the components would only increase unnecessary nesting as I am pretty sure we might never use it.

Reply:
Do you think we can reuse it in the seller parking spot list? Currently, what we have is checkbox, would this be a good idea?